docs/geedocs/5.2.0/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title>Google Earth Enterprise Documentation</title>
+<link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" />
```

Comment:
Sorry, found one more problem.  This page doesn't load the CSS correctly when I install it locally.  I think you need to take out the '..' in front of the two css paths.